recipes/libint2/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -x
+echo ${PREFIX}
+export CXXFLAGS="${CXXFLAGS} -O2 -pipe -march=x86-64 -std=c++11 -fPIC -fopenmp"
+export CPPFLAGS="-I${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib"
```

Comment:
I think `CPPFLAGS` and `LDFLAGS` here can both be dropped. Both are included in the default compiler flags, which we don't want to override.